app/javascript/components/molecules/MultiSelect.js

```diff
@@ -21,42 +21,57 @@ const getStyleOverrides = (controlHeight) => {
   return {
     control: (base, _) => ({
       ...base,
-      height: controlHeightPx
+      minHeight: controlHeightPx
+    }),
+    multiValue: (base, _) => ({
+      ...base,
+      maxWidth: '90%'
     })
   }
 }
 
 const MultiSelect = ({
-  items = [],
-  selectedItems = [],
+  input = null,
+  options = [],
+  value = [],
   height = HEIGHT_NORMAL,
   disabled = false,
-  onChangeValues = () => {}
-}) => (
-  <div className='input-custom-height'>
-    <Select
-      isDisabled={disabled}
-      isClearable
-      isSearchable
-      isMulti
-      styles={getStyleOverrides(height)}
-      onChange={(values) => onChangeValues(values)}
-      value={selectedItems}
-      options={items}
-    />
-  </div>
-)
+  onChange = () => {}
+}) => {
+  let selectedValues
+  if (value && value[0] && !value[0].value) {
+    selectedValues = options.filter((option) => value.includes(option.value))
```

Comment:
I see now what you're doing to parse the string input. I think this is a fragile way of handling inputs because if your options list is like:
Children_Ages:
[
'1',
'2',
'Over 2'
]

And your `value` prop == `'1, Over 2'`, this code will think '1', '2', and 'Over 2' are selected.

I think it's better to either enforce a specific delimiter type in this component(e.g., `', '`), or do the conversion to SelectedItemShape before passing into this component. I slightly prefer the latter since I don't love one prop that can have multiple forms

Reply:
not sure I follow. Component returns an array of strings, so it's formatted nicely for the API. But then react select expects {value, label} object. That functions searches options by value to find the correct one. I don't see a room for error here.

Reply:
Value can be `'1'` or `['1', 'Over 2']`

Reply:
Interesting, okay, so it's never a string if it's multiple values?

I still think the issue occurs if the value is `'Over 2'`. In that case `selectedValues` would be ['2', 'Over 2'] since both match with includes.

It should work with `value === option.value` as the check instead. The only thing is now this definitely doesn't work if multiple values are passed in a single comma-delimited string (which is fine if that's never happening